### PR TITLE
Indigo devel

### PR DESCRIPTION
**Pull request by LucidOne**

### CODE REVIEW

**Comment by wjwwood** [comment]
`catkin_prepare_release` should be used for doing this...

File: pcl_ros/CHANGELOG.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Changelog for package pcl_ros
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1.1.8 (2014-04-09)
+1.2.0 (2014-04-09)
```
Comment (wjwwood) [inline pcl_ros/CHANGELOG.rst:5]: This is almost certainly not what you intended to do...

**Comment by wjwwood** [comment]
I highly recommend you use the tools and workflows described in this section of this tutorial:

http://wiki.ros.org/bloom/Tutorials/ReleaseCatkinPackage#bloom.2BAC8-Tutorials.2BAC8-PrepareUpstream.Preparing_the_Upstream_Repository

**Comment by LucidOne** [comment]
I'll try again shortly.

```
user@host:~/catkin_ws/src/perception_pcl/perception_pcl$ catkin_generate_changelog
Could not detect repository type - currently supports: git, hg
```

So I thought it needed to be fixed manually as I updated the package.xml and thought it was pulling from there.
`<url type="repository">https://github.com/ros-perception/perception_pcl</url>`

I now understand

```
user@host:~/catkin_ws/src/perception_pcl$ catkin_generate_changelog
Found packages: pcl_ros, perception_pcl
Querying commit information since latest tag...
Updating forthcoming section of changelog files...
- updating 'pcl_ros/CHANGELOG.rst'
- updating 'perception_pcl/CHANGELOG.rst'
Done.
Please review the extracted commit messages and consolidate the changelog entries before committing the files!
```

**Pull request merged**